_includes/components/kofi.njk

```diff
@@ -0,0 +1,14 @@
+{# Ko-Fi marco #}
+{% from "./svgEmoji.njk" import svgEmoji %}
+
+{# NOTE: AFAIK, metadata is not accessible from here so the Ko-Fi social URL has to be passed in from the parent #}
+{% macro kofi(url, item = "post") %}
+<article class="kofi" aria-labelledby="kofi-title">
+  <span id="kofi-title" hidden>Donate</span>
+  <p>
+    If you enjoyed this {{ item }}, please feel free to
+    <a href={{url}}>buy me a coffee</a>!
+    {{ svgEmoji("☕️") }}
```

Comment:
VoiceOver seems to ignore `aria-hidden="true"` on inline SVGs with text that are a descendant of paragraphs, so might as well just use a `<span>` with `aria-hidden="true"` here because that seems to work with other screen readers like NVDA.